Panic if give wrong raw key

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x11cb6a3]

goroutine 20 [running]:
testing.tRunner.func1(0xc4200871e0)
        /Users/mtk10671/.gvm/gos/go1.8.1/src/testing/testing.go:622 +0x29d
panic(0x12e83a0, 0x14bf050)
        /Users/mtk10671/.gvm/gos/go1.8.1/src/runtime/panic.go:489 +0x2cf
github.com/appleboy/easyssh-proxy/vendor/golang.org/x/crypto/ssh.publicKeyCallback.auth(0xc42010f720, 0xc42010ffa0, 0x14, 0
x20, 0x13439f3, 0x9, 0x1494360, 0xc4200d4b00, 0x148fb60, 0xc420082bd0, ...)
        /Users/mtk10671/git/go/src/github.com/appleboy/easyssh-proxy/vendor/golang.org/x/crypto/ssh/client_auth.go:193 +0xb
3
```

--- a/easyssh.go
+++ b/easyssh.go
@@ -86,8 +86,9 @@ func getSSHConfig(config DefaultConfig) *ssh.ClientConfig {
 	}
 
 	if config.Key != "" {
-		signer, _ := ssh.ParsePrivateKey([]byte(config.Key))
-		auths = append(auths, ssh.PublicKeys(signer))
+		if signer, err := ssh.ParsePrivateKey([]byte(config.Key)); err == nil {
+			auths = append(auths, ssh.PublicKeys(signer))
+		}
 	}
 
 	return &ssh.ClientConfig{

--- a/easyssh_test.go
+++ b/easyssh_test.go
@@ -284,3 +284,19 @@ func TestSCPCommandWithPassword(t *testing.T) {
 		t.Fatalf("SCP-error: %v", err)
 	}
 }
+
+func TestWrongRawKey(t *testing.T) {
+	// wrong key
+	ssh := &MakeConfig{
+		Server: "localhost",
+		User:   "drone-scp",
+		Port:   "22",
+		Key:    "appleboy",
+	}
+
+	outStr, errStr, isTimeout, err := ssh.Run("whoami", 10)
+	assert.Equal(t, "", outStr)
+	assert.Equal(t, "", errStr)
+	assert.False(t, isTimeout)
+	assert.Error(t, err)
+}

